client/me/profile-links-add-wordpress/index.jsx

```diff
@@ -171,6 +171,7 @@ class ProfileLinksAddWordPress extends Component {
 						{
 							components: {
 								jetpackLink: (
+									/* eslint-disable-next-line jsx-a11y/anchor-is-valid */
```

Comment:
I'd suggest removing this. It's a valid ESLint warning that I don't see a good reason to suppress.

Reply:
The `a` element should probably be a `button`. Do we have a convenient way to style `button` so that it looks like an inline hyperlink?

Alternatively, it could remain an `a` element, but specify `href="http://jetpack.me"`, with a `_target` attribute to open a new window. And the `a` element can still have an `onClick` handler that records the tracks events and calls `this.onCancel()`.

Reply:
Thanks for the suggestion, @jsnajdr
Fixed in https://github.com/Automattic/wp-calypso/pull/48350/commits/6fc709cf058c649ac94d00937553e6d8e9675d45

Reply:
FWIW: `Button` from `@automattic/components` has a `plain` which will reset all styles, making it easy to style it as an anchor. But probably better is to use `@wordpress/components` `Button` which has a `isLink` prop.

Reply:
> But probably better is to use @wordpress/components Button which has a isLink prop.

I tried that but the styling doesn't seem right.
<img width="279" alt="Screenshot 2020-12-16 at 17 50 54" src="https://user-images.githubusercontent.com/14192054/102372417-d9cc0380-3fc7-11eb-809f-ec16c59d6090.png">
I'll leave the `a` element with `href` and `onClick` attributes which seems to be the cleanest solution that also makes eslint happy.